tests/cql/CqlComparisonOperatorsTest.xml

```diff
@@ -25,7 +25,7 @@
 			<output>false</output>
 		</test>
 		<test name="SimpleEqNullNull">
-			<expression>null = null</expression>
+			<expression>null as String = null</expression>
```

Comment:
In this and the following two fixes, the translator complains because it cannot resolve a call to Equal w/ System.Any on both sides.  To avoid this, we can cast null as any concrete type and the translator is then happy.